pkg/util/release/environment_test.go

```diff
@@ -0,0 +1,54 @@
+package release
+
+import (
+	"testing"
+
+	shipper "github.com/bookingcom/shipper/pkg/apis/shipper/v1alpha1"
+)
+
+func TestIsReleaseProgressing(t *testing.T) {
+	var tests = []struct {
+		title        string
+		achievedStep *shipper.AchievedStep
+		targetStep   int32
+		expected     bool
+	}{
+		{
+			"nil achieved step",
+			nil,
+			0,
+			true,
+		},
+		{
+			"empty achieved step",
+			&shipper.AchievedStep{},
+			0,
+			true,
+		},
+		{
+			"progressing release",
+			&shipper.AchievedStep{
+				Step: 0,
+				Name: "0",
+			},
+			2,
+			true,
+		},
+		{
+			"not progressing release",
+			&shipper.AchievedStep{
+				Step: 1,
+				Name: "1",
+			},
+			0,
+			false,
+		},
```

Comment:
Could you add a test for the equality? (achieved step == targetStep) As it has been a confusion in the comments of the code, I think clearly stating the intent of the code in tests could alleviate some doubt from the reader.